### PR TITLE
ecs_certificate: allow to request renewal without csr

### PR DESCRIPTION
**Pull request by francescolovecchio**
##### SUMMARY
See [specification](https://cloud.entrust.net/EntrustCloud/documentation/cms-api-2.1.0.yaml) and [module documentation](https://docs.ansible.com/ansible/latest/collections/community/crypto/ecs_certificate_module.html#parameter-csr)

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
ecs_certificate

##### ADDITIONAL INFORMATION
Renew certificate requests without csr failed with:

```
The csr field is required when request_type=renew
```

### CODE REVIEW

**Comment by felixfontein** [comment]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by felixfontein** [comment]
CC @ctrufan

**Comment by francescolovecchio** [comment]
@felixfontein  changelog fragment added! Thanks

File: changelogs/fragments/740-ecs_certificate-renewal-without-csr
Comment (felixfontein) [inline changelogs/fragments/740-ecs_certificate-renewal-without-csr:1]: You need to add the `.yml` extension to the filename :)

Comment (francescolovecchio) [reply changelogs/fragments/740-ecs_certificate-renewal-without-csr:1]: Sure, thank you

**Comment by felixfontein** [comment]
@sapnajainEntrust do you know whom to contact from Entrust's side to review this? @ctrufan doesn't seem to be active on GitHub anymore.

**Comment by ctrufan** [comment]
Sorry about the delay. Looks good!

**Review by ctrufan — APPROVED** [review]
Changes are consistent with (newer versions of) Entrust API specification.

**Comment by felixfontein** [comment]
@francescolovecchio thanks for this fix!
@ctrufan thanks for reviewing! good to see you again :)

**Pull request merged**